src/Raven.Client/Documents/Operations/Backups/GetPeriodicBackupStatusOperation.cs

```diff
@@ -45,11 +47,19 @@ public override void SetResponse(JsonOperationContext context, BlittableJsonRead
                     ThrowInvalidResponse();
 
                 Result = JsonDeserializationClient.GetPeriodicBackupStatusOperationResult(response);
+                if (Result.IsSharded)
+                    throw new InvalidOperationException($"Database is sharded, can't use {nameof(GetPeriodicBackupStatusOperation)}, " +
+                                                        $"use {nameof(GetShardedPeriodicBackupStatusOperation)} instead");
             }
         }
     }
 
-    public class GetPeriodicBackupStatusOperationResult
+    public abstract class AbstractGetPeriodicBackupStatusOperationResult
+    {
+        public bool IsSharded;
```

Comment:
can you mark it as internal?

Reply:
if I mark it as internal then `JsonDeserializationClient.GetShardedPeriodicBackupStatusOperationResult(response)` ignores it and  `IsSharded` is always `false`